modules/installation-restricted-network-samples.adoc

```diff
@@ -26,11 +45,22 @@ not addressed in this procedure.
 
 .Procedure
 
+. Access a specific imagestream to mirror, for example:
```

Comment:
@gabemontero the updates are merged and I've been working in conditionals for the restricted network-specific content and the standard configuring the samples operator-specific content and came up with a few more questions. (I'll put links to the built versions in too, because that makes it much easier to read.) 

For the configuring the samples operator-specific procedure, do we need to provide instructions for mirroring? Or is mirroring not relevant if someone is just looking to use a different registry? And if the procedures are relevant, but the verbiage is awkward, should I edit the language around mirroring for this purpose?

Reply:
Thanks! I reused the instructions and updated the configuring the samples operator content. It might eventually need to be it's own procedure, but we'll keep it there for now.